src/app/visual/action-hub/action-hub-visual.component.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-action-hub-visual',
+  templateUrl: './action-hub-visual.component.html'
+})
+export class ActionHubVisualComponent {
+  /* istanbul ignore next */
+  public needsAttention = [
+    {
+      title: '1 update',
+      message: 'from portal',
+      permalink: {url: '#'}
+    },
+    {
+      title: '2 new messages',
+      message: 'from online donation',
+      permalink: {url: '#'}
+    },
+    {
+      title: '3 possible duplicates',
+      message: 'from constituent lists',
+      permalink: {url: '#'}
+    },
+    {
+      title: '4 updates',
+      message: 'from portal',
+      permalink: {url: '#'}
+    },
+    {
+      title: '5 new messages',
+      message: 'from online donation',
+      permalink: {url: '#'}
+    },
+    {
+      title: '6 possible duplicates',
+      message: 'from constituent lists',
+      permalink: {url: '#'}
+    },
+    {
+      title: '7 update',
+      message: 'from portal',
+      permalink: {url: '#'}
+    },
+    {
+      title: '8 new messages',
+      message: 'from online donation',
+      permalink: {url: '#'}
+    },
+    {
+      title: '9 possible duplicates',
+      message: 'from constituent lists',
+      permalink: {url: '#'}
+    }
+  ];
+  /* istanbul ignore next */
+  public recentLinks = [
+    {
+      label: 'Recent 1',
```

Comment:
Can we format the two arrays here the same? I like how the second is formatted but it is jarring that we space the permalink on the second but not the first.